scapy/contrib/gtp_v2.py

```diff
@@ -646,8 +699,7 @@ class IE_AMBR(gtp.IE_Base):
 class IE_MSISDN(gtp.IE_Base):
     name = "IE MSISDN"
     fields_desc = [ByteEnumField("ietype", 76, IEType),
-                   FieldLenField("length", None, length_of="digits",
-                                 adjust=lambda pkt, x: x + 4, fmt="H"),
+                   ShortField("length", None),
```

Comment:
Why are you changing this value?

Reply:
Same comment as - https://github.com/secdev/scapy/pull/2488#discussion_r386762044